src/scss/partials/variables/bootstrap/_variable-overrides.scss

```diff
@@ -13,7 +13,10 @@ $gray-700: #697180;
 $gray-800: #363a41;
 $gray-900: #232323;
 
-
+$header-bg: #fff;
+$header-top-bg: $header-bg;
+$header-color: $gray-800;
+$header-top-color: $header-color;
```

Comment:
These variables shouldn't live there, They should be from the `prestashop` folder, inside a `_header` partial

Reply:
Changed to partial _header.scss